test/cypress/plugins/utilities.js

```diff
@@ -0,0 +1,18 @@
+export function changeTimezone(date, tz) {
+  // suppose the date is 12:00 UTC
+  var invdate = new Date(date.toLocaleString('en-US', {
+    timeZone: tz
+  }));
+
+  // then invdate will be 07:00 in Toronto
+  // and the diff is 5 hours
+  var diff = date.getTime() - invdate.getTime();
+
+  // so 12:00 in Toronto is 17:00 UTC
+  return new Date(date.getTime() + diff);
+};
+
+export function zeroPadding(input) {
```

Comment:
Fear of using left-pad 🤣

Reply:
not sure if you're asking about the function name or the new string padding function which I've never tried? 😝 
it seems I could use this instead `str.padStart(2, '0');`

Reply:
My joke did not catch fire. If you remember the world wide npm issue because of npm left-pad, thats what I was referring to ;)

But yeah other than that padStart certainly od better. Less code = less bugs